cmd/threatest/run.go

```diff
@@ -0,0 +1,221 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/datadog/threatest/pkg/threatest"
+	"github.com/datadog/threatest/pkg/threatest/parser"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"math"
+	"os"
+	"strconv"
+	"time"
+)
+
+// RunCommand implements the command to run Threatest test scenarios
+type RunCommand struct {
+	SSHConfig      *SSHConfiguration
+	InputFiles     []string
+	Parallelism    int
+	JsonOutputFile string
+}
+
+type SSHConfiguration struct {
+	SSHHost     string
+	SSHUsername string
+	SSHKey      string
+}
+
+type ScenarioRunResult struct {
+	Description     string    `json:"description"`
+	Success         bool      `json:"isSuccess"`
+	ErrorMessage    string    `json:"errorMessage"`
+	DurationSeconds float64   `json:"durationSeconds"`
```

Comment:
mostly because it's annoying to serialize IIRC. Typically when it's marshalled to JSON, I wanted to avoid the extra complexity of having a custom marshmaller that converts to seconds.

Unless you have a strong opinion I'll leave as is for the sake of simplicity